blockly-core/core/ui/contract_editor/contract_editor.js

```diff
@@ -177,27 +177,27 @@ Blockly.ContractEditor.prototype.create_ = function() {
   this.exampleAreaDiv.style.position = 'absolute';
   goog.dom.insertChildAt(this.container_, this.exampleAreaDiv, 0);
 
-  var callText = goog.dom.createDom('div', 'callText');
-  callText.innerHTML = "Call";
-  goog.dom.appendChild(this.exampleAreaDiv, callText);
+  this.callText = goog.dom.createDom('div', 'callText');
```

Comment:
As you mentioned in HipChat, it does feel like we might want to pull this and other Example-section-specific state out into an e.g. `ExampleSectionView`, (maybe?) inheriting from `ContractEditorSectionView`, just to make the stored DOM element reference boundaries clearer.

A client of such an ExampleSectionView might want to then register `onRun` and `onReset` callbacks, so the ExampleSectionView doesn't need to know about the business logic of running examples, but just triggering them and displaying results (i.e., we should try to limit passed references to that).